__tests__/app.test.js

```diff
@@ -40,6 +41,41 @@ describe("API: /api/categories", () => {
 });
 
 describe("API: /api/reviews", () => {
+  describe("GET: /api/reviews", () => {
+    test("200: responds with array of reviews objects containing the properties of, describe and slug  ", () => {
```

Comment:
This test description isn't very clear because you don't specify all of the properties. Keeping it brief eg `200: responds with array of reviews objects` would be much clearer.

Reply:
Thanks Haz, I will implemet it now!!